src/main/java/org/isf/dicom/manager/SourceFiles.java

```diff
@@ -375,18 +376,18 @@ public static synchronized void loadDicom(FileDicom dicomFileDetail, File source
 				studyUID = dicomObject.getString(Tag.StudyInstanceUID) == null ? studyUID : dicomObject.getString(Tag.StudyInstanceUID);
 				accessionNumber = dicomObject.getString(Tag.AccessionNumber) == null ? accessionNumber : dicomObject.getString(Tag.AccessionNumber);
 				studyDescription = dicomObject.getString(Tag.StudyDescription) == null ? studyDescription : dicomObject.getString(Tag.StudyDescription);
-				String studyComments = dicomObject.getString(Tag.StudyComments);
+				//String studyComments = dicomObject.getString(Tag.StudyComments);
```

Comment:
> Why these comments?

and this

Reply:
Because they are variables that aren't used in the method; but I left them there in case someone knew why they were included.    I am more than willing to remove them.